CoreScriptsRoot/CoreScripts/Topbar.lua

```diff
@@ -802,44 +853,33 @@ local function CreateSettingsIcon(topBarInstance)
 	return menuItem
 end
 
-local function Create3DSettingsIcon(topBarInstance, panel)
-	local menuItem = CreateSettingsIcon(topBarInstance)
-
-	rawset(menuItem, "Hover", function(self, hovering)
-		if hovering then
-			self:SetImage("rbxasset://textures/ui/Menu/HamburgerDown.png")
-		else
-			self:SetImage("rbxasset://textures/ui/Menu/Hamburger.png")
-		end
-	end)
+local function Create3DSettingsIcon(topBarInstance, panel, menubar)
+	local MenuModule = nil
```

Comment:
I think we should move the local down a few lines to where we initialize it, I don't see why we need for forward declare this variable.